addons/mail/static/src/discuss/call/common/call_action_list.xml

```diff
@@ -99,4 +60,21 @@
         </div>
     </t>
 
+    <t t-name="discuss.CallActionList.actionButton">
+        <button class="btn d-flex m-1 border-0 rounded-circle shadow-none opacity-100 opacity-75-hover"
+            t-att-class="{ 'p-2': isSmall, 'p-3': !isSmall }"
+            t-att-aria-label="action.name"
+            t-att-title="action.name"
+            t-on-click="action.select">
+            <div class="fa-stack">
```

Comment:
I would expect "Join call"/"Disconnect" (and even mobile "Push-To-Talk", though very custom design) to also be call actions. But not a huge deal for now.

To be put after more, we could use negative sequences. The advantage of them being also call actions is for code in `call_action_list` being even simpler, and all expected call actions are fully defined in call action registry.

I don't mind this being the case for now ;)